contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -1,16 +1,227 @@
 #!/bin/bash
 
+# This script is used to start the zetacored nodes
+# It initializes the nodes and creates the genesis.json file
+# It also starts the nodes
+# The number of nodes is passed as an first argument to the script
+# The second argument is optional and can have the following value:
+# 1. upgrade : This is used to test the upgrade process, a proposal is created for the upgrade and the nodes are started using cosmovisor
+
+/usr/sbin/sshd
+
+if [ $# -lt 1 ]
+then
+  echo "Usage: genesis.sh <num of nodes> [option]"
+  exit 1
+fi
+NUMOFNODES=$1
+OPTION=$2
+
+# create keys
+CHAINID="athens_101-1"
+KEYRING="test"
```

Comment:
In ZetaClient we have this option because we wanted to test our encryption logic for the hotkey with use with the client.
For the `zetacored` we didn't need to use a different keyring so far, might be an idea to explore in another PR if it makes sense but we don't add our own custom logic to test on top of the SDK keyring here, unlike ZetaClient